roles/federation-registry/tasks/postgres.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Make sure psycopg2 is installed
+  command: pip install psycopg2==2.7.5 --ignore-installed
+
+- name: Send Create Tables SQL File
+  copy:
+    src: private_files/setup_db.sql
+    dest: "{{federation_registry_db.script_path}}"
+
+- name: Create a new database with name service_registry_db
+  postgresql_db:
+    name: service_registry_db
+  become: yes
+  become_user: postgres
+
```

Comment:
Remove. DB creation will be handled externally (e.g. with rciam/postgres role or the db is provided externally